src/5.0_to_5.1/migrateUserActivity.test.ts

```diff
@@ -0,0 +1,23 @@
+import { migrateUserActivity } from "./migrateUserActivity";
+import { contentServer } from "./__test_resources__/contentServer";
+import {
+  dualCatalogDataModel,
+  sandboxDataModel,
+} from "@activeviam/data-model-5.1/dist/__test_resources__";
+
+const dataModels = {
+  sandbox: sandboxDataModel,
+  dualCatalog: dualCatalogDataModel,
+};
+
+describe("migrateUserActivity", () => {
+  it("migrates 5.0 `userFilters` and `userQueryContext` to usable data in 5.1", () => {
+    migrateUserActivity(contentServer, dataModels);
```

Comment:
I would add a snapshot of the content before calling `migrateUserActivity` too:
- it's a good illustration of what the function does. When reading the test, one clearly sees the state of the content before the mutation, and after
- it ensures the data from the centralized `contentServer` is not corrupted. It's probably not the scariest example, but let's say someone modifies `user2`'s query context's value from `30` (number) to `"30"` (string), the test would still pass whereas we don't check anymore that the value was stringified